python/ray/experimental/channel/nccl_group.py

```diff
@@ -0,0 +1,84 @@
+from typing import Dict, Optional
+
+import ray
+from ray.util.collective.collective_group import nccl_util
+
+try:
+    import torch
+except ImportError:
+    torch = None
+
+
+class _NcclGroup:
```

Comment:
nit;
```suggestion
class NcclGroup:
```
IIUC, _ means we cannot use it in other module

Reply:
I think that's okay?